test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,79 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class DefaultIncludeTest < ActiveSupport::TestCase
+      class Blog < ActiveModelSerializers::Model
+        attr_accessor :id, :name, :posts
+      end
+      class Post < ActiveModelSerializers::Model
+        attr_accessor :id, :title, :author, :category
+      end
+      class Author < ActiveModelSerializers::Model
+        attr_accessor :id, :name
+      end
+      class Category < ActiveModelSerializers::Model
+        attr_accessor :id, :name
+      end
+
+      class BlogSerializer < ActiveModel::Serializer
+        default_include 'posts.author'
+        attributes :id
+        attribute :name, key: :title
+
+        has_many :posts
+      end
+      class PostSerializer < ActiveModel::Serializer
+        default_include 'author'
+        attributes :id, :title
+
+        has_one :author
+        has_one :category
+      end
+      class AuthorSerializer < ActiveModel::Serializer
+        attributes :id, :name
+      end
+      class CategorySerializer < ActiveModel::Serializer
+        attributes :id, :name
+      end
+
+      setup do
+        @authors = [Author.new(id: 1, name: 'Blog Author')]
+        @categories = [Category.new(id: 1, name: 'Food')]
+        @posts = [
+            Post.new(id: 1, title: 'The first post', author: @authors.first, category: @categories.first),
+            Post.new(id: 2, title: 'The second post', author: @authors.first, category: @categories.first)]
+        @blog = Blog.new(id: 2, name: 'The Blog', posts: @posts)
+      end
+
+      test '#default_include option populate "included" for json_api adapter' do
+        serialized = ActiveModelSerializers::SerializableResource.new(@blog, serializer: BlogSerializer, adapter: :json_api).as_json
+
+        assert_equal serialized[:included].size, 3
```

Comment:
for both these tests and the always tests, I'd actually make sure that the correct types are rendered, just so whomever is looking at these tests in the future can very easily see _what_ is being rendered.